src/dump1090.c

```diff
@@ -4198,16 +4201,45 @@ static bool set_home_pos (const char *arg)
 
   if (arg)
   {
-    if (sscanf(arg, "%lf,%lf", &pos.lat, &pos.lon) != 2 || !VALID_POS(pos))
+    if (sscanf(arg, "%lf,%lf", &pos.lat, &pos.lon) != 2)
     {
       LOG_STDERR ("Invalid home-pos %s.\n", arg);
       return (false);
     }
+    
+    // if -1,-1 - launch setup
+    if (pos.lat == -1.0 && pos.lon == -1.0)
+    {
+        char exePath[MAX_PATH];
+        char dirPath[MAX_PATH];
+        char setupPath[MAX_PATH];
+
+        if (GetModuleFileNameA(NULL, exePath, MAX_PATH))
+        {
+            strcpy_s(dirPath, sizeof(dirPath), exePath);
+            PathRemoveFileSpecA(dirPath);
+            PathCombineA(setupPath, dirPath, "setup.exe");

```

Comment:
This could be simpler:
`snprintf (exePath, sizeof(exePath), "%s\\setup.exe", Modes.where_am_I);`

No need for another OS library.